src/OAuth2Client.php

```
@@ -86,37 +101,126 @@
     public function request(string $method, string $url, array $options = []): Response
     {
         $this->ensureValidToken();
 
         try {
-            $response = Http::withToken($this->accessToken)
-                ->send($method, $url, $options);
+            $http = Http::withToken($this->accessToken)
+                ->acceptJson();
+
+            // Disable SSL verification if configured
+            if ($this->shouldDisableSSLVerification()) {
```

PHPStan (2.1.15)

Call to an undefined method Antogkou\LaravelOAuth2Client\OAuth2Client::shouldDisableSSLVerification().


(method.notFound)